example/privacy/main.go

```diff
@@ -0,0 +1,279 @@
+
+	flag.Parse()
+
+	client, err := qlcchain.NewQLCClient(flagNodeUrl)
+	if err != nil || client == nil {
+		fmt.Println(err)
+		return
+	}
+
+	fmt.Println(client.Version())
+
+	demoContractAbi(client)
+
+	//demoMintageContract(client)
+
+	//demoMinerContract(client)
+
+	demoPrivacyDemoKVContract(client)
+}
+
+func demoContractAbi(client *qlcchain.QLCClient) {
+	fmt.Println("============ demoContractAbi ============")
+	rspAddrs := client.Contract.ContractAddressList()
+	fmt.Printf("ContractAddressList:\n%s\n", util.ToIndentString(rspAddrs))
+
+	for _, ca := range rspAddrs {
+		rspAbi, err := client.Contract.GetAbiByContractAddress(ca)
+		if err != nil {
+			fmt.Println("GetAbiByContractAddress", err)
+			return
+		}
+		fmt.Printf("GetAbiByContractAddress:\n%s, %s\n", ca, rspAbi)
+	}
+}
+
+func demoMintageContract(client *qlcchain.QLCClient) {
+	fmt.Println("============ demoMintageContract ============")
+	/*
+			{
+		    "type":"function",
+		    "name":"Mintage",
+		    "inputs":[
+		        {
+		            "name":"tokenId",
+		            "type":"tokenId"
+		        },
+		        {
+		            "name":"tokenName",
+		            "type":"string"
+		        },
+		        {
+		            "name":"tokenSymbol",
+		            "type":"string"
+		        },
+		        {
+		            "name":"totalSupply",
+		            "type":"uint256"
+		        },
+		        {
+		            "name":"decimals",
+		            "type":"uint8"
+		        },
+		        {
+		            "name":"beneficial",
+		            "type":"address"
+		        },
+		        {
+		            "name":"NEP5TxId",
+		            "type":"string"
+		        }
+		    ]
+		}
+	*/
+	minerAddr, _ := types.HexToAddress("qlc_3hw8s1zubhxsykfsq5x7kh6eyibas9j3ga86ixd7pnqwes1cmt9mqqrngap4")
+	minerSendPara := qlcchain.MintageParams{
+		SelfAddr:    minerAddr,
+		TokenName:   "QBTC",
+		TokenSymbol: "QBTC",
+		TotalSupply: "2100000000000000",
+		Decimals:    8,
+		Beneficial:  minerAddr,
+		NEP5TxId:    util.RandomFixedString(32),
+	}
+	mintRspData, err := client.Mintage.GetMintageData(&minerSendPara)
+	if err != nil {
+		fmt.Println("GetMintageData", err)
+		return
+	}
+	fmt.Printf("GetMintageData:\n%s\n", hex.EncodeToString(mintRspData))
+
+	mintRspBlk, err := client.Mintage.GetMintageBlock(&minerSendPara)
+	if err != nil {
+		fmt.Println("GetMintageBlock", err)
+		return
+	}
+	fmt.Printf("GetMintageBlock:\n%s\n", util.ToIndentString(mintRspBlk))
+
+	contractSendPara := qlcchain.ContractSendBlockPara{
+		Address:   minerSendPara.SelfAddr,
+		To:        types.MintageAddress,
+		TokenName: "QLC",
+		Amount:    types.NewBalance(5 * 1e13),
+		Data:      mintRspData,
+	}
+	contractRspBlk, err := client.Contract.GenerateSendBlock(&contractSendPara)
+	if err != nil {
+		fmt.Println("GenerateSendBlock", err)
+		return
+	}
+	fmt.Printf("GenerateSendBlock:\n%s\n", util.ToIndentString(contractRspBlk))
+}
+
+func demoMinerContract(client *qlcchain.QLCClient) {
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
func `demoMinerContract` is unused (unused)